pkg/consensus/consensus.go

```diff
@@ -268,7 +268,9 @@ func (s *service) Name() string {
 func (s *service) Start() {
 	if s.started.CAS(false, true) {
 		s.log.Info("starting consensus service")
-		s.dbft.Start()
+		b, _ := s.Chain.GetBlock(s.Chain.CurrentBlockHash()) // Can't fail, we have some current block!
+		s.lastTimestamp = b.Timestamp * uint64(time.Millisecond)
```

Comment:
```suggestion
		s.lastTimestamp = b.Timestamp
```
See https://github.com/nspcc-dev/neo-go/blob/0f625f04f0046f3f8d8a4d9c8411cfc72cfd5293/pkg/consensus/consensus.go#L571

Reply:
Yep, I've outsmarted myself here.